docker/generate_makefile.py

```diff
@@ -56,16 +59,28 @@ def _print_makefile_run_template(image):
 \t-e FUZZ_TARGET=$({benchmark}-fuzz-target) \\\
 '.format(fuzzer=fuzzer, benchmark=benchmark))
 
+        params = ""
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
             print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+        elif run_type == 'repro-check':
+            print('\t-v {path}:/testcases \\\n\
+\t--entrypoint /bin/bash '.format(path=bench_path),
+                  end='')
+            params = " -c \"for f in /testcases/*; do\
```

Comment:
Can this be not printed at the same time ?

Reply:
No, it needs to be printed after the image name, which is done after the ifs for every rule type